app/serializers/webhook_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class WebhookSerializer < ActiveModel::Serializer
+  attributes :text, :username, :attachments, :icon_url
```

Comment:
We might need to have different serializers for different targets (hipchat, trello, whatever), but since it's just slack at the moment the one will do I think